micro-ui/web/packages/modules/workbench/src/components/BulkUploadNew.js

```
@@ -0,0 +1,205 @@
+  };
+
+  const renderFileCards = useMemo(() => {
+    return fileData?.map((file, index) => (
+      <div
+        className="uploaded-file-container"
+        key={index}
+        onClick={(e) => {
+          e.stopPropagation();
+          setShowPreview(true);
+        }}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)